lib/objectid.js

```diff
@@ -0,0 +1,5 @@
+var ObjectId = require('mongodb').ObjectId;
```

Comment:
I don't think we should depend on `mongodb` directly in the juggler. We should contribute `ObjectID` some other way. It should work without the `mongodb` connector (eg. in the browser).